x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/inference.tsx

```diff
@@ -158,7 +148,8 @@ const fieldsConfig: FieldsConfig = {
 
 export const Inference: FunctionComponent = () => {
   const { services } = useKibana();
-  const esDocUrl = services.documentation.getEsDocsBasePath();
+  const esDocRegression = services.documentation.getRegressionUrl();
```

Comment:
Can we also name these `regressionDocsLink` and `classificationDocsLink` for consistency with the above suggestion?